src/runtime_src/core/common/system.h

```diff
@@ -11,6 +11,10 @@
 // 3rd Party Library - Include files
 #include <boost/property_tree/ptree.hpp>
 
+#ifdef __linux__
+#pragma GCC diagnostic ignored "-Woverloaded-virtual"
+#endif
```

Comment:
Not in header file, that disables for all compilation units that end up including this file.